docs/HowToAddnewBoard.md

```diff
@@ -0,0 +1,108 @@
+# How to add(port) a new board
+
+Changes in *[arch](#arch)* and *[configs](#configs)* folders are mandantory to support a new board.  
+Adding static library or expanding driver will be provided later.
+
```

Comment:
Addition of New Drivers into os/driver model ,
Getting the Wi-Fi driver up and hooking with the wpa_supplicant can be a part of separate doc linked to this for refrence